Signup/Signup.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://airave-ui.netlify.app/component.css" rel="stylesheet">
+    <title>Airave Mart</title>
+</head>
+<body>
+    <nav>
+        <div class="nav-brand ml-5">
+            <div class="nav-logo">Logo</div> 
+                <a href="#"><p class="brand">Airave</p></a>
```

Comment:
Inside a tag can be tags with default display: inline or inline-block. tags like span, em, strong etc. check this